lib/heap/index.js

```diff
@@ -25,9 +24,18 @@ var Heap = module.exports = integration('Heap')
  */
 
 Heap.prototype.initialize = function(page){
-  window.heap=window.heap||[];window.heap.load=function(a){window._heapid=a;var d=function(a){return function(){window.heap.push([a].concat(Array.prototype.slice.call(arguments,0)));};},e=["identify","track"];for (var f=0;f<e.length;f++)window.heap[e[f]]=d(e[f]);};
-  window.heap.load(this.options.appId);
-  this.load(this.ready);
+  window.heap = window.heap || [];
+  window.heap.appid = this.options.appId;
+
+  var o = function(t){
+    return function(){
+      window.heap.push([t].concat(Array.prototype.slice.call(arguments, 0)));
+    };
+  };
+  p = ["identify", "track"];
+  for (c = 0; c < p.length; c++) window.heap[p[c]] = o(p[c]);
```

Comment:
Nit: could we change these to real variable names? Just makes it overall easier to read :)